inc/builder/type/header/mode-switcher/dynamic-css/dynamic.css.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Mode Switcher - Dynamic CSS
+ *
+ * @package Astra
+ * @since x.x.x
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Mode Switcher
+ */
+add_filter( 'astra_dynamic_theme_css', 'astra_mode_switcher_dynamic_css' );
+
+/**
+ * Dynamic CSS
+ *
+ * @param  string $dynamic_css          Astra Dynamic CSS.
+ * @param  string $dynamic_css_filtered Astra Dynamic CSS Filters.
+ * @return String Generated dynamic CSS for Mode Switcher.
+ *
+ * @since x.x.x
+ */
+function astra_mode_switcher_dynamic_css( $dynamic_css, $dynamic_css_filtered = '' ) {
+
+	if ( ! Astra_Builder_Helper::is_component_loaded( 'mode-switcher', 'header' ) ) {
+		return $dynamic_css;
+	}
+
+	$_section = 'section-mode-switcher';
+	$selector = '.ast-header-mode-switcher-wrap';
+
+	$icon_size         = astra_get_option( 'mode-switcher-icon-size' );
+	$icon_size_desktop = ( ! empty( $icon_size['desktop'] ) ) ? $icon_size['desktop'] : 20;
+	$icon_size_tablet  = ( ! empty( $icon_size['tablet'] ) ) ? $icon_size['tablet'] : 20;
+	$icon_size_mobile  = ( ! empty( $icon_size['mobile'] ) ) ? $icon_size['mobile'] : 20;
+
+	/**
+	 * Mode Switcher - Desktop CSS.
+	 */
+	$css_output_desktop = array(
+		$selector . ' svg' => array(
+			'height' => astra_get_css_value( $icon_size_desktop, 'px' ),
+			'width'  => astra_get_css_value( $icon_size_desktop, 'px' ),
+		),
```

Comment:
```suggestion
		'#svg' => array(
			'height' => astra_get_css_value( $icon_size_desktop, 'px' ),
			'width'  => astra_get_css_value( $icon_size_desktop, 'px' ),
		),
```
@imnavanath Can we directly target to the class/ID of the SVG instead of multiple selectors?

Reply:
Great suggession, thanks. I'll implement this.

Reply:
Updated likewise 👍 Added `ast-mode-switcher-icon` class to those 4 SVGs and updated it in CSS.